WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartFullScreenDialogFragment.java

```diff
@@ -125,4 +126,16 @@ public void onSaveInstanceState(@NonNull Bundle outState) {
             outState.putBoolean(KEY_COMPLETED_TASKS_LIST_EXPANDED, mQuickStartAdapter.isCompletedTasksListExpanded());
         }
     }
+
+    @Override
+    public void onSkipTaskTapped(QuickStartTask task) {
+        mQuickStartStore.setDoneTask(AppPrefs.getSelectedSite(), task, true);
```

Comment:
```suggestion
        // TODO: Quick Start - Add analytics for skipping task.
        mQuickStartStore.setDoneTask(AppPrefs.getSelectedSite(), task, true);
```

We will want to track when an individual task is skipped.  Can we add a `// TODO` here so we don't forget?